services/payments/tests/unit/test_services_payments_gateway.py

```diff
@@ -60,15 +97,19 @@ async def test_payment_gateway_responsiveness(
     assert await payment_gateway_api.is_healhy()
 
 
+@pytest.mark.testit
```

Comment:
forgotten `@pytest.mark.testit`

Reply:
I will remove in next PR .noe everything is green :-)

Reply:
ye np :) i would recommend to force merge :+1: